pylib/anki/exporting.py

```diff
@@ -21,9 +21,18 @@
 class Exporter:
     includeHTML: Union[bool, None] = None
 
-    def __init__(self, col: _Collection, did: None = None) -> None:
+    did: Optional[int]
+    cids: Optional[List[int]]
```

Comment:
If my understanding is correct, if an argument to `__init__` specifies a type, it is not necessary to also define it at the class level. The cases where you see types specified at the class level in current code are usually cases where __init__ is untyped, or where the instance variable is being set outside of `__init__`

Reply:
Sorry if I wasn't clear - it'd be better to keep the type for cid in the `__init__` call, and remove the did/cid definitions from the class, as the constructor arg should be enough.

Otherwise looks good, thanks.

Reply:
Corrected